src/v/redpanda/admin/api-doc/config.json

```diff
@@ -79,4 +79,30 @@
       }
     }
   }
-}
+},
+"/v1/config/blocked_reactor_notify_ms/{timeout}": {
```

Comment:
timeout should be in the body, or a query parameter. having it be a path parameter is a non-intuitive API since timeout isn't like an addressable resource, it's a property of the reactor_notify resource.